Apps/RestServer/src/main/java/rapture/server/rest/RestServer.java

```diff
@@ -236,9 +239,125 @@ private void setupRoutes() {
             }
             return Kernel.getDecision().createWorkOrder(getContext(req), getWorkorderUriParam(req), params);
         });
-
+        
+        post("/sstore/:authority", (req, res) -> {
+        	log.info(req.body());
+            Map<String, Object> data = JacksonUtil.getMapFromJson(req.body());
+            String authority = req.params(":authority");
+            String config = (String) data.get("config");
+            CallingContext ctx = getContext(req);
+            if (Kernel.getStructured().structuredRepoExists(ctx, authority)) {
+                halt(409, String.format("Repo [%s] already exists", authority));
+            }
+            Kernel.getStructured().createStructuredRepo(ctx, authority, config);
+            return new RaptureURI(authority, Scheme.STRUCTURED).toString();
+        });
+        
+        post("/sstore/:authority/:table", (req, res) -> {
+        	log.info(req.body());
+        	Map<String, Object> data = JacksonUtil.getMapFromJson(req.body());
```

Comment:
indentation off.  I recommend auto-formatting on save.

Go to Window->Preferences->Java->Editor->SaveActions and check the Format source code option. Then your source code will be formatted automatically each time you save it.